lib/main_common.pm

```diff
@@ -586,6 +581,8 @@ sub load_jeos_tests {
         if (is_sle) {
             loadtest "console/suseconnect_scc";
         }
+
+        replace_opensuse_repos_tests if is_repo_replacement_required;
```

Comment:
This is inside the `unless (get_var('LTP_COMMAND_FILE'))` block. Does the LTP testsuite not need to install packages? It would've been broken before anyway though...

Reply:
Refreshed repos would be required. Therefore, I have put it on the same level as `suseconnect_scc`. Seems like we can copy the flow from osd and rework this part. I will create a follow-up ticket.